src/what-unsafe-does.md

```diff
@@ -51,8 +50,10 @@ points to are part of the same allocation (so in particular they all have to be
 part of *some* allocation). The span of bytes it points to is determined by the
 pointer value and the size of the pointee type. As a consequence, if the span is
 empty, "dangling" is the same as "non-null". Note that slices point to their
-entire range, so it's very important that the length metadata is never too
-large. If for some reason this is too cumbersome, consider using raw pointers.
+entire range, so it's very important that the length metadata is never too large
```

Comment:
```suggestion
entire range, so it's important that the length metadata is never too large
```

Reply:
FWIW I think "very" is much more in line with the style of the Nomicon. But I am too lazy to argue...